en/config/actuators.md

```diff
@@ -109,31 +146,104 @@ Selecting these requires a reboot before they are applied.
+   2. For Fixed Wings, the thrust should point to forward direction
+   3. For VTOL, thrust should point upwards when the Tilt Servo is at 0 degrees as defined the [Tilt Servo Convention](#motor-tilt-servos-conceptual). Testing of the [Tilt Servo](#tilt-servo-testing) is covered below as well.
+
+#### Control Surface setup
+
+![Control Surface Disarmed 1500 Setting](../../assets/config/actuators/control_surface_disarmed_1500.png)
+
+1. **Set the 'Disarmed' value for Control Surface outputs to '1500'** in case of using a PWM output
+   * Unlike Motors, when **disarmed** we want the control surface to be in the middle commanded value, which correlates to 1500 in PWM
+2. **Set the Output mode to 'PWM 50Hz'**, instead of default PWM 400 Hz when using a PWM output for Control Surfaces using Servos
+   * If your Servo is creating weird noise, it is most likely because of the "PWM 400Hz" output, which is usually not accepted by commercial Servos.
+   * In case your Servo supports 100, 200 or 400 Hz command, you can of course set the output mode to the maximum PWM frequency it supports!
+3. Move Control Surface slider upwards (positive command) and verify that it moves in the direction defined in the [Control Surface Convention](#control-surface-direction-conceptual).
+   1. If the control surface moves in the opposite direction, click on the 'Rev Range' checkbox to reverse the range
+4. Move the slider to the middle and check if the Control Surfaces are aligned in the neutral position of the wing
+    ![Control Surface Trimming](../../assets/config/actuators/control_surface_trim.png)
+   1. If it is not aligned, you can set the **Trim** value for each Control Surface.
+   2. After setting a new trim, manually click and place the slider for the control surface you are testing in the middle position, to check if it is in the neutral position.
+   3. Note that as mentioned above as a safety feature of slider, unless you actually touch the slider, the actuator command doesn't get sent. Therefore **you must manually click and put the slider in the middle position to test the trim**
+   4. Or, an easier way to test without using the slider would be to set the [`COM_PREARM_MODE`](../advanced_config/parameter_reference.md#COM_PREARM_MODE) parameter to `Always`.
+      1. Which will enable the control of Servos even when the vehicle is disarmed, and will constantly be applying the Trim setting to the Control Surfaces
+      2. Here you can try setting different values for the Trim and check the alignment, and then settle on the value you are happy with.
+
+#### Tilt Servo setup
+![Tilt Servo Setup](../../assets/config/actuators/tilt_servo_setup.png)
+
+> Setup of Tilt Servo is similar to Control Surface, as both of them uses the Servo hardware (in most cases)
+
+1. **Set the 'Disarmed' value for Control Surface outputs to '1500'** in case of using a PWM output
```

Comment:
This depends on the vtol, some might want to use 1000 or 2000.

Reply:
Thanks @bkueng. I changed this to as shown below. If this is not OK let me know (since this text may be moved).

```suggestion
1. **Set the 'Disarmed' value for control surface outputs. The value depends on the vehicle.
```

Are there any particular "considerations" for this? I mean for a _failsafe_ value on an FW vehicle it is not uncommon to set the value such that the vehicle does a slow glide or circle to land. For an MC motor you want a _disarmed_ value to not power a motor. This is kind of FMI, but perhaps might be useful to provide additional hints above.